_posts/articles/2022-08-16-genart.md

```diff
@@ -0,0 +1,28 @@
+
+# AI Village Art Day @ DEF CON 30
+Today, models that appear to generate novel outputs conditioned on text are becoming the mainstay of popular culture.
+New groups of users are starting to adopt generative architectures and bring new modes (experiential, creative) of interacting with large models.
+The understanding of model performance is becoming increasingly qualitative and subjective, as is evident in the latest benchmarks for text-to-image models.
```

Comment:
Are you trying to say that non-technical users are now forming opinions on models and model performance, or are you trying to say that ML researchers disagree on what metrics should be used for benchmarking? Either way, more detail would be helpful!

Reply:
yeah, great point of clarification -- I would basically aim at showing that there is now a plurality of ways of engaging with the models, and some (those practiced by artists / those posting to Twitter) are more experiential; in ways, it is helpful to researchers (who may have a different modality in which to appreciate all the new models) to understand how the model is being used out there _and_ perhaps suggest new models for incentivizing exploration of unintended / notable behavior that is reliant on humans, rather than more principled exploration (but somehow, hopefully, can suggest novel approaches)

if nothing else - "it looks pretty" and "inductive bias over many iterations" crowds talking to each other could be entertaining, if nothing else (and hopefully no theology this time around)